fe/src/main/java/org/apache/doris/alter/SchemaChangeJobV2.java

```diff
@@ -953,4 +969,131 @@ public void readFields(DataInput in) throws IOException {
             }
         }
     }
+
+    /**
+     * write data need to persist when job finished
+     */
+    private void writeJobFinishedData(DataOutput out) throws IOException {
+        // only persist data will be used in getInfo
+        out.writeInt(indexIdMap.size());
```

Comment:
Write double?

Reply:
why use writeDouble? indexIdMap.size() is a integer type.

Reply:
Ignore me. I yesterday looked the code write `out.writeInt(indexIdMap.size());` twice. So I said write double.